### ISSUE TITLE
'The app is invalid.' error when sideload an ipa file.

### ISSUE THREAD

**Issue by ShadyWhite** [rileytestut/AltStore#244]
**Describe the bug**
When I sideload an ipa file,I got 'The app is invalid.' error.The app's display name on iphone includes Chinese words.I tried to modify the 'Info.plist' file to change the app's name to English but no use.

**To Reproduce**
Use Altstore to sideload this .ipa file.https://drive.google.com/file/d/1JzeJF1YiAgyEdPUuvft0rv6qbBJUGInh/view?usp=sharing

**Expected behavior**
I expected to sideload this .ipa file on my iPhone.

**Screenshots**
![IMG_0011](https://user-images.githubusercontent.com/44015352/83733455-21b8b800-a680-11ea-87a9-60f64a24f001.PNG)

**Desktop (please complete the following information if applicable):**
 - OS: [Windows]
 - Version: [Windows 10]

**iPhone (please complete the following information):**
 - Device: [iPhoneX]
 - iOS: [13.5]

**Comment by Yolkersuu** [comment]
I've got the same issue with an other api file I try to sideload

**Comment by tanmx** [comment]
I've got the same issue

**Comment by Markizzzzz** [comment]
Is it going to be patched any time soon?

**Comment by Lampadina17** [comment]
Still even on 14.4.2

**Comment by zeratax** [comment]
this happens for me as well, specifically with https://github.com/EhPanda-Team/EhPanda/blob/main/AltStore.json

I am on 16.1.1 on AltStore 1.5.1

I wish there was literally any info on this ^^

related: https://github.com/EhPanda-Team/EhPanda/issues/264

**Comment by zeratax** [comment]
tried out the beta but error messages there aren’t any more helpful 
![alt store beta saying in both the error message and failure reason field: "the app is invalid."](https://user-images.githubusercontent.com/5024958/203149551-d585867d-c727-42f4-9541-8a862f6a6fe4.png)

**Comment by shanegillio** [comment]
we've marked this as an issue with special character app names - there are other similar github issue reports. I'm just going to close this as a duplicate

**Comment by zeratax** [comment]
> we've marked this as an issue with special character app names - there are other similar github issue reports. I'm just going to close this as a duplicate

the app in question only has ascii characters afaict? i am confused

**Comment by shanegillio** [comment]
You are correct - it may be something in the app's JSON source file. I'mm reopen and mark this to look at for reference
